src/features/typeDecorations.ts

```diff
@@ -38,9 +59,10 @@ export default () => {
                     return content
                 })
                 .join('')
-            const typeMatch = /: (.+)/.exec(hoverString)
+            // eslint-disable-next-line no-await-in-loop
+            const typeMatch = (await checkIfInStyles(document, pos)) ? /Syntax: (.*)/.exec(hoverString) : /: (.+)/.exec(hoverString)
```

Comment:
@ILScc why did you supress `no-await-in-loop`? You actually could actually done this check before loop. This is not optimal.